app/views/status/YourRegistrationView.scala.html

```diff
@@ -72,7 +72,7 @@ <h2 id="status-submitted-business" class="govuk-heading-s">@messages("status.bus
             }
         }
         </div>
-        <div id="registration-status" class="govuk-grid-column-one-third">
+        <div id="registration-status" class="govuk-grid-column-one-third">Deregister your business
```

Comment:
should the deregister your business text be here?

Reply:
no , accidental copy paste